datumaro/plugins/data_formats/datumaro/base.py

```diff
@@ -281,21 +282,3 @@ def _load_annotations(item):
                 raise NotImplementedError()
 
         return loaded
-
-
-class DatumaroImporter(Importer):
-    @classmethod
-    def detect(cls, context: FormatDetectionContext) -> None:
-        annot_file = context.require_file("annotations/*.json")
-
-        with context.probe_text_file(
-            annot_file,
-            'must be a JSON object with "categories" ' 'and "items" keys',
-        ) as f:
-            contents = parse_json(f.read())
-            if not {"categories", "items"} <= contents.keys():
-                raise Exception
-
-    @classmethod
-    def find_sources(cls, path):
-        return cls._find_sources_recursive(path, ".json", "datumaro", dirname="annotations")
```

Comment:
This seems like the duplication of https://github.com/openvinotoolkit/datumaro/pull/824.